community-operators/dynatrace/dynatrace-monitoring.v0.2.0.clusterseviceversion.yaml

```diff
@@ -0,0 +1,239 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [{
+      	"apiVersion": "dynatrace.com/v1alpha1",
+      	"kind": "OneAgent",
+      	"metadata": {
+      		"name": "oneagent",
+      		"namespace": "dynatrace"
+      	},
+      	"spec": {
+      		"apiUrl": "https://ENVIRONMENTID.live.dynatrace.com/api",
+      		"skipCertCheck": false,
+      		"tokens": "",
+      		"nodeSelector": {},
+      		"tolerations": [
+      			{
+      				"effect": "NoSchedule",
+      				"key": "node-role.kubernetes.io/master",
+      				"operator": "Exists"
+      			}
+      		],
+      		"image": "",
+      		"args": [
+      			"APP_LOG_CONTENT_ACCESS=1"
+      		],
+      		"env": []
+      	}
+      }]
+    categories: Monitoring, OpenShift Optional
```

Comment:
Shouldn't this be a String of comma separated values?

Reply:
Added double quotes